Dockerfile

```diff
@@ -466,12 +464,10 @@ ARG COMPOSER_VERSION=2.2
 RUN curl -ksS https://getcomposer.org/installer | php -- --install-dir=/bin --filename=composer --$COMPOSER_VERSION
 
 # ORT
-COPY --chown=$USERNAME:$USERNAME --from=ortbuild /opt/ort /opt/ort
-COPY docker/ort-wrapper.sh /usr/bin/ort
-COPY docker/ort-wrapper.sh /usr/bin/orth
-RUN chmod 755 /usr/bin/ort
+COPY --from=ort /opt/ort /opt/ort
+ENV PATH=$PATH:/opt/ort/bin
```

Comment:
IIUC correctly from previous commits, `orth` is under `/opt/ort/bin`.
Have you double checked whether it's possible to run the `helper-cli` by just typing `orth` in bash inside this docker container?

Reply:
Yep, the whole /opt/ort/bin is in the PATH now

Reply:
Thx!